src/ImageSharp/Formats/Tiff/PhotometricInterpretation/Rgb888TiffColor.cs

```diff
@@ -24,6 +25,7 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
 
             int offset = 0;
 
+            var rgba = default(Rgba32);
```

Comment:
same here, use `L8`

Reply:
it seems rgba is needed here, 3 components are used

Reply:
yes you are right, sorry